sdk-src/src/main/java/com/hpe/adm/nga/sdk/Octane.java

```diff
@@ -156,12 +158,15 @@ public void signOut() {
      * Once the correct context has been built up use the {@link #build()} method to create the <code>Octane</code> instance
      */
     public static class Builder {
+
+        public static final String OCTANE_HTTP_CLIENT_CLASS_NAME = "octaneHttpClientClassName";
+
         //Private
         private final Logger logger = LogManager.getLogger(Octane.class.getName());
-        private String urlDomain = "";
-        private String idsharedSpaceId = null;
-        private long workSpaceId = 0;
-        private final Authentication authentication;
+        protected String urlDomain = "";
+        protected String idsharedSpaceId = null;
+        protected long workSpaceId = 0;
+        protected final Authentication authentication;
 
```

Comment:
@slbruce I think it's leftovers, from when I tried extending the builder, will change it back to private.